bin/create_tls_key.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-set -xe
+set -e
 
-mkdir -p ~/.irslackd
-openssl req -newkey rsa:4096 -nodes -sha512 -x509 -days 3650 -nodes -out ~/.irslackd/cert.pem -keyout ~/.irslackd/pkey.pem
-fingerprint=$(openssl x509 -noout -fingerprint -sha512 -inform pem -in ~/.irslackd/cert.pem | cut -d= -f2-)
+while getopts ":d:" opt; do
+  case $opt in
+    d) dir="$OPTARG"
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&2
+    ;;
+  esac
+done
+if [ -z "$dir" ]; then
+   dir="~/.irslackd"
+fi
```

Comment:
Can set `dir="~/.irslackd"` at the top of the script instead of this conditional.